spec/integration/integration_spec.rb

```diff
@@ -0,0 +1,190 @@
+    visit("http://localhost:3000/users/#{tom.id}/")
+    link = find_link('See all posts', href: "/users/#{tom.id}/posts")
+    expect(link).not_to be_nil
+  end
+
+  it 'When a post is clicked it redirects to that post page' do
+    visit("http://localhost:3000/users/#{tom.id}/")
+    click_link(post.title)
+    expect(page).to have_current_path "http://localhost:3000/users/#{tom.id}/posts/#{post.id}"
+  end
+
+  it 'When See all posts is clicked it redirects to the users\' posts page' do
+    visit("http://localhost:3000/users/#{tom.id}/")
+    click_link('See all posts')
+    expect(page).to have_current_path "http://localhost:3000/users/#{tom.id}/posts"
+  end
+end
+
+RSpec.describe 'Posts page.', type: :system do
+  subject!(:tom) { User.where(name: 'Tom').first }
+
+  before(:all) do
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'Shows username' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Tom')
+  end
+
+  it 'Shows profile picture' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    image = page.all('img')
+    expect(image[0]['src']).to eq('https://acolombo1.github.io/Portfolio/foto1.jpg')
+  end
+
+  it 'Shows the number of posts the user has written.' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Number of posts: 4')
+  end
+
+  it 'Shows a posts title' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Hello')
+  end
+
+  it 'Shows some of the posts body' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('This is my first post')
+  end
+
+  it 'Shows the first comments on a post' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Hi, how are you?')
+  end
+
+  it 'Shows how many comments a post has' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Comments: 6')
+  end
+
+  it 'Shows how many likes a post has' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Likes: 0')
+  end
+
+  it 'Shows button for pagination if there are more posts than fit on the view' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    expect(page).to have_content('Pagination')
+  end
+
+  it 'When a post is clicked, redirects to that post\'s show page' do
+    visit("http://localhost:3000/users/#{tom.id}/posts")
+    click_link('Hello')
+    expect(page).to have_content('Hello by Tom')
+  end
+end
+
+RSpec.describe 'Posts page.', type: :system do
+  subject!(:tom) { User.where(name: 'Tom').first }
+  subject!(:post) { Post.where(author_id: 1).first }
+
+  before(:all) do
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'Shows posts title' do
+    visit("http://localhost:3000/users/#{tom.id}/posts/#{post.id}")
+    expect(page).to have_content('Hello by Tom')
+  end
```

Comment:
- Some of your tests are failing, kindly fix them to proceed to the next project 😄
![Screenshot from 2022-08-23 22-35-16](https://user-images.githubusercontent.com/44624138/186271274-5a007b6d-2bc4-42f0-a391-1c9520055c49.png)

Reply:
It's the first test in each describe. None of these happen in my computer or my partner's. What can we do? Can you show the failure reason?, all those <unknown> don't tell anything. I can look for a workaround maybe, like having an empty 'it' in each describe.